sleap/gui/dialogs/formbuilder.py

```diff
@@ -437,13 +460,22 @@ def add_item(self, item: Dict[Text, Any]):
 
         # double: show spinbox (number w/ up/down controls)
         elif item["type"] == "double":
-            field = QtWidgets.QDoubleSpinBox()
+            
+            #If the label of the item is Loss Weight then we will set the min and max to 0 and 1000
+            if item["label"] == "Loss Weight":
+                field = ExponentialSpinBox()
+                min = 0
+                max = 1000
+                field.setDecimals(2)
+            else:
+                field = QtWidgets.QDoubleSpinBox()
+                min, max = -1000, 1000
+                field.setSingleStep(0.25)
 
-            min, max = -1000, 1000
             if "range" in item.keys():
                 min, max = list(map(float, item["range"].split(",")))
             field.setRange(min, max)
-            field.setSingleStep(0.25)
+                
```

Comment:
Running `black` on this file will format this for us - either through vscode's black extension + `formatOnSave` setting or through the CLI:

```bash
black  sleap/gui/dialogs/formbuilder.py
```
.
```suggestion
```